packages/block-editor/src/store/reducer.js

```diff
@@ -1913,6 +1913,14 @@ export function blockEditingModes( state = new Map(), action ) {
 	return state;
 }
 
+export function openedBlockSettingsMenu( state = null, action ) {
+	switch ( action.type ) {
+		case 'SET_OPENED_BLOCK_SETTINGS_MENU':
+			return action.clientId || null;
+	}
```

Comment:
```suggestion
	if ( 'SET_OPENED_BLOCK_SETTINGS_MENU' === action.type ) {
		return action.clientId || null;
	}
```

Reply:
Just checking it's okay to remove the `null` since the return value of the [selector](https://github.com/WordPress/gutenberg/pull/54083/files/c3b5092d7972d879be61f89ee701cc3e55601d6f) is `undefined`.

Reply:
Oh, yes, `undefined` should be fine — I think `null` is just a leftover from where I copy/pasted. I'll have a quick play and then commit your suggestion!